src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs

```diff
@@ -380,8 +380,12 @@ internal static LoggerConfiguration FromLogContext(LoggerEnrichmentConfiguration
             => loggerEnrichmentConfiguration.FromLogContext();
 
         // Unlike the other configuration methods, Logger is an instance method rather than an extension.
-        internal static LoggerConfiguration Logger(LoggerSinkConfiguration loggerSinkConfiguration, Action<LoggerConfiguration> configureLogger, LoggingLevelSwitch restrictedToMinimumLevel = null)
-            => loggerSinkConfiguration.Logger(configureLogger, levelSwitch: restrictedToMinimumLevel);
```

Comment:
`levelSwitch: restrictedToMinimumLevel` is the cause of the chaos.

Reply:
oh man :-/ sorry about thus one

Reply:
Easy to miss! (I did when reviewing it, anyway! :-))